packages/web-components/src/components/va-file-input/va-file-input.tsx

```diff
@@ -89,6 +89,12 @@ export class VaFileInput {
    */
   @Prop() headerSize?: number;
 
+  /**
+   * DST only prop
+   * removes extraneous display for multiple file input
+   */
+  @Prop() headless?: boolean = false;
+
```

Comment:
Needed to add a prop that tells it not to render everything so multiple file input looks correct